index.js

```diff
@@ -4,14 +4,24 @@ const { AutoDeployApi } = require("./autodeploy");
 async function run() {
   let status;
   try {
-    const pods = core.getInput("pods", { required: true });
+    const pods = core.getInput("pods");
+    const images = core.getInput("images");
     const tag = core.getInput("tag") || "develop";
     const url = core.getInput("url", { required: true });
     const token = core.getInput("token", { required: true });
 
     core.info(`Re-deploying pods ${pods} for tag ${tag}`);
     const api = new AutoDeployApi(url, token);
-    status = await api.redeploy(pods, tag);
+
+    if (pods) {
+      status = await api.redeploy("services", tag);
+    }
+    if (images) {
+      status = await api.redeploy("images", tag);
```

Comment:
The new signature is `AutoDeploy.redeploy(type, targets, pod)` which seems to be used here incorrectly now.